### PR TITLE
Mindy Allen- Project 1

### PR DESCRIPTION
**Pull request by MAllen07**

### CODE REVIEW

File: queries.md [hunk 1]
```diff
@@ -2,16 +2,28 @@
...
 
+select \* from customers where city = 'London'
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+select \* from customers where postalcode = '1010'
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+select \* from suppliers where supplierid = 11
```
Comment (Jorlost19) [inline queries.md]: Here the instruction says you need to find the phone number and you are searching instead for all. needs to be fixed.

File: server.js [hunk 2]
```diff
@@ -2,6 +2,11 @@ const express = require('express');
 
 const server = express();
 
+Foobar.get('data/accounts-model.js', (req, res) => {
```
Comment (Jorlost19) [inline server.js]: Foobar? LOL

File: server.js [hunk 3]
```diff
@@ -2,6 +2,11 @@ const express = require('express');
...
+    return res.send('access to new database')
+}
+)
+
```
Comment (Jorlost19) [inline server.js:12]: Still waiting on those endpoints to be done.

File: queries.md (hunk 1 above)
Comment (MAllen07) [reply queries.md]: made changes-
select phone from suppliers where supplierId = 11

File: server.js (hunk 3 above)
Comment (MAllen07) [reply server.js:12]: end points added

**Review by JamieHall1962 — COMMENTED** [review]
## MVP

### Queries

-[x]  find all customers that live in London. Returns 6 records.
-[x]  find all customers with postal code 1010. Returns 3 customers.
-[x]  find the phone number for the supplier with the id 11. Should be (010) 9984510.
-[x]  list orders descending by the order date. The order with date 1997-02-12 should be at the top.
-[x]  find all suppliers who have names longer than 20 characters. You can use length(SupplierName) to get the length of the name. Returns 11 records.
-[x]  find all customers that include the word "market" in the name. Should return 4 records.
-[x]  add a customer record for "The Shire", the contact name is "Bilbo Baggins" the address is "1 Hobbit-Hole" in "Bag End", postal code "111" and the country is "Middle Earth".
-[x]  update Bilbo Baggins record so that the postal code changes to "11122".

### Endpoints

-[ ]  GET /api/accounts
-[ ]  GET /api/accounts/:id
-[ ]  POST /api/accounts
-[ ]  PUT /api/accounts/:id
-[ ]  DELETE /api/accounts/:id

### Stretch

-[ ]  list orders grouped by customer showing the number of orders per customer. Rattlesnake Canyon Grocery should have 7 orders.
-[ ]  list customers names and the number of orders per customer. Sort the list by number of orders in descending order. Ernst Handel should be at the top with 10 orders followed by QUICK-Stop, Rattlesnake Canyon Grocery and Wartian Herkku with 7 orders each.
-[x]  list orders grouped by customer's city showing number of orders per city. Returns 58 Records with Aachen showing 2 orders and Albuquerque showing 7 orders.
-[x]  delete all customers that have no orders. Should delete 18 records.

1.5 Stars

* Nice job on the MVP queries
* Good start on the stretch queries. 2 work, 1 does not, and 1 was not attempted
* Multiple problems on the server. If you fix it, please push the changes, let me know, and I will update your frade
* You attempt to import and use helmet and morgan but they are not included in your package.json. Use yarn add

**Pull request merged**